src/components/sbb-form-field/sbb-form-field.tsx

```diff
@@ -0,0 +1,180 @@
+import { Component, Element, h, Prop, State } from '@stencil/core';
+import { InterfaceSbbFormFieldAttributes } from './sbb-form-field.custom';
+import { AgnosticMutationObserver as MutationObserver } from '../../global/helpers/mutation-observer';
+
+let nextId = 0;
+
+/**
+ * @slot label - Slot to render a label.
+ * @slot prefix - Slot to render an icon on the left side of the input.
+ * @slot input - Slot to render an input.
+ * @slot suffix - Slot to render an icon on ther right side of the input.
+ * @slot error - Slot to render an error.
+ */
+@Component({
+  shadow: true,
+  styleUrl: './sbb-form-field.scss',
+  tag: 'sbb-form-field',
+})
+export class SbbFormField {
+  /**
+   * Add a specific space if the `<sbb-error>` is present.
+   */
+  @Prop() public errorSpace?: InterfaceSbbFormFieldAttributes['errorSpace'] = 'default';
+
+  /**
+   * Add a `<label>` for the input.
+   */
+  @Prop() public label: string;
+
+  /**
+   * Indicates whether the input is optional.
+   */
+  @Prop() public optional?: boolean;
+
+  /**
+   * Size variant, either l or m.
+   */
+  @Prop() public size?: InterfaceSbbFormFieldAttributes['size'] = 'l';
+
+  /**
+   * @internal
+   * It is used internally to get the native `readonly` attribute from `<input>`.
+   */
+  @State() private _readonly: boolean;
+
+  /**
+   * @internal
+   * It is used internally to get the native `disabled` attribute from `<input>`.
+   */
+  @State() private _disabled: boolean;
+
+  /**
+   * The internal `<sbb-form-field>` element.
+   */
+  @Element() private _element: HTMLElement;
+
+  /**
+   * @internal
+   * It is used internally to set the `for` attribute of the `<label>`.
+   */
+  private _id: string;
+
+  /**
+   * @internal
+   * It is used internally to set the `aria-describedby` attribute into the `input` whether the `<sbb-form-error>` is present.
+   */
+  private _idError: string;
+
+  /**
+   * @internal
+   * It is used internally to get the `input` slot.
+   */
+  private _input: HTMLInputElement;
+
+  /**
+   * @internal
+   * Listen the changes on `readonly` and `disabled` attributes of `<input>`.
+   */
+  private _formFieldAttributeObserver = new MutationObserver(this._onAttributesChange.bind(this));
+
+  public componentWillLoad(): void {
+    this._idError = this._element?.querySelector('[slot="error"]')?.getAttribute('id');
```

Comment:
The ids of the errors (there might be multiple), should be evaluated dynamically by listening to the `slotchange` event of the error slot.